Netconf-notif-v12-examples-v17Sept2018/ietf-subscribed-notifications.yang

```diff
@@ -635,7 +636,7 @@ module ietf-subscribed-notifications {
     output {
       leaf id {
         type subscription-id;
-        mandatory true;
+        //mandatory true;
```

Comment:
Probably `yanglint` bug as with an error the `output` clause probably shouldn't apply. Check RFCs?

Reply:
Same here.

Reply:
I am not sure how else this could be modeled.   A successful RPC response of this type must have a subscription-id.